storybook/src/components/icon-button/icon-button.core-template.ts

```diff
@@ -0,0 +1,23 @@
+import { IconButton } from "dso-toolkit";
+import { html } from "lit-html";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+
+import { ComponentImplementation } from "../../templates";
+
+export const coreIconButton: ComponentImplementation<IconButton> = {
+  component: "iconButton",
+  implementation: "core",
+  template: () =>
+    function iconButtonTemplate({ accessibleLabel, dsoIconButtonClick, disabled, icon, tooltipPlacement, variant }) {
+      return html`
+        <dso-icon-button
+          variant=${variant}
+          icon=${icon}
+          accessible-label=${accessibleLabel}
+          tooltip-placement=${ifDefined(tooltipPlacement)}
+          disabled=${ifDefined(disabled)}
```

Comment:
`disabled` is een boolean attribute, die zetten we met de `?` operator van lit (en zonder `ifDefined()`):


```suggestion
          ?disabled=${disabled}
```

Hiermee voorkomen we `disabled="false"`.

Reply:
sweet